### ISSUE TITLE
Pass a dom node to `render` instead of a selector string

### ISSUE THREAD

**Issue by colinmeinke** [colinmeinke/wilderness#65]
There is no reason Wilderness needs to be doing `querySelector` work. It can just be passed a DOM node instead.

```js
render({ selector: 'svg' }, circle)
```

Becomes:

```js
const el = document.querySelector('svg')
render({ el }, circle)
```

**Comment by colinmeinke** [comment]
Same goes for `shape` function.

```js
shape({ selector: 'path' })
```

Becomes:

```js
const el = document.querySelector('path')
shape({ el })
```

**Comment by colinmeinke** [comment]
Closed by cbc2211676db693d3659326912e939ba53ff4b2f
